### ISSUE TITLE
IPA fails to upload. APK is not zip aligned?

### ISSUE THREAD

**Issue by kNoAPP** [wzieba/Firebase-Distribution-Github-Action#56]
I am using this GitHub action to upload appbundles & ipas to Firebase for Android/iOS. The android appbundle uploads without issue. The ipa fails with the error below. I don't believe it to be anything wrong with my ipa because I can download the artifact my GitHub Action generates (`output.ipa`) and manually upload it to Firebase without error.

What's stranger is that this error seems to apply only to `.apk`s. As this was generated upon uploading an `.ipa`.

What are my next steps in debugging?

```
Run wzieba/Firebase-Distribution-Github-Action@v1.3.2
  with:
    appId: ***
    token: ***
    file: output.ipa
    releaseNotes: ### Added
  - iOS Build Action support
    groups: internal
    debug: false
/usr/bin/docker run --name a717d81b949eaf49668084d58d9caf6e89_f38b9e --label 9916a7 --workdir /github/workspace --rm -e INPUT_APPID -e INPUT_TOKEN -e INPUT_FILE -e INPUT_RELEASENOTES -e INPUT_GROUPS -e INPUT_SERVICECREDENTIALSFILE -e INPUT_RELEASENOTESFILE -e INPUT_DEBUG -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/hylist/hylist":"/github/workspace" 9916a7:17d81b949eaf49668084d58d9caf6e89
sh: Added: unknown operand
i  uploading binary...

Error: failed to upload release. There's been an error processing your app because of an archiving issue. Use zipalign in your build process and try again. Learn more at https://developer.android.com/studio/command-line/zipalign If this problem continues, contact Firebase support.
Additional information about this error:
APK is not zip aligned.
```

**Comment by mateuszkwiecinski** [comment]
>sh: Added: unknown operand

@kNoAPP it looks like there is something wrong with `releaseNotes` input. I don't know if it can be escaped on the action side, but you can try to escape it on your own by wrapping the release notes in a quotes:
```
releaseNotes: "### Added"
```

**Comment by kNoAPP** [comment]
Right, my bad, I forgot to post an update. I noticed the release notes issue too. While the release notes were making it to Firebase just fine, it was a little strange what that output was doing. These release notes were escaped.

So I just converted the notes to a file and that made the action happy. But it did not solve the problem.

What did solve the problem is not using the Android appId for the Apple upload. I didn't stop to think the Android appId and Apple appId are different. Making this change solved the issue.
